### PR TITLE
Feature/request images

### PR DESCRIPTION
**Pull request by dominikkukacka**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/Jumio/con-rest/pull/73%23issuecomment-87972414%22%2C%20%22https%3A//github.com/Jumio/con-rest/pull/73%23issuecomment-94712119%22%2C%20%22https%3A//github.com/Jumio/con-rest/pull/73%23issuecomment-103475785%22%2C%20%22https%3A//github.com/Jumio/con-rest/pull/73%23discussion_r29734837%22%2C%20%22https%3A//github.com/Jumio/con-rest/pull/73%23discussion_r29734983%22%2C%20%22https%3A//github.com/Jumio/con-rest/pull/73%23discussion_r29735080%22%2C%20%22https%3A//github.com/Jumio/con-rest/pull/73%23discussion_r30572119%22%2C%20%22https%3A//github.com/Jumio/con-rest/pull/73%23discussion_r30575290%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/Jumio/con-rest/pull/73%23issuecomment-87972414%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22code%20review%20%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222015-03-31T07%3A21%3A11Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1508400%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/EnoF%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-04-21T09%3A01%3A55Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8627051%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/TheBay0r%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-05-19T12%3A56%3A42Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1508400%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/EnoF%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20268468bd3a04209ac6e2dc2af2cb06f67029e350%20callback.js%2020%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/Jumio/con-rest/pull/73%23discussion_r29734983%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22I%27ll%20clean%20this%20up%20in%20an%20additional%20commit%22%2C%20%22created_at%22%3A%20%222015-05-06T05%3A18%3A31Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1508400%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/EnoF%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20callback.js%3AL1-37%22%7D%2C%20%22Pull%20268468bd3a04209ac6e2dc2af2cb06f67029e350%20callback.js%201%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/Jumio/con-rest/pull/73%23discussion_r29735080%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22What%20is%20the%20use%20of%20an%20additional%20server%3F%20%22%2C%20%22created_at%22%3A%20%222015-05-06T05%3A23%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1508400%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/EnoF%22%7D%7D%2C%20%7B%22body%22%3A%20%22that%27s%20just%20for%20testing%20the%20execution%20of%20requests%20against%20an%20endpoint%22%2C%20%22created_at%22%3A%20%222015-05-19T06%3A29%3A34Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/243319%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dominikkukacka%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20callback.js%3AL1-37%22%7D%2C%20%22Pull%20268468bd3a04209ac6e2dc2af2cb06f67029e350%20app/viewModels/fileC.js%201%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/Jumio/con-rest/pull/73%23discussion_r29734837%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22typo%2C%20I%27ll%20amend%20the%20commit%22%2C%20%22created_at%22%3A%20%222015-05-06T05%3A12%3A36Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1508400%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/EnoF%22%7D%7D%2C%20%7B%22body%22%3A%20%22already%20did%20that%20%3A%29%22%2C%20%22created_at%22%3A%20%222015-05-19T07%3A36%3A02Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/243319%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dominikkukacka%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20app/viewModels/fileC.js%3AL1-57%22%7D%7D%2C%20%22approved%22%3A%20%7B%22https%3A//github.com/EnoF%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1508400%3Fv%3D3%22%7D%2C%20%22https%3A//github.com/TheBay0r%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8627051%3Fv%3D3%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/EnoF'></a><a href='https://github.com/TheBay0r'></a>

- [ ] <a href='#crh-comment-Pull 268468bd3a04209ac6e2dc2af2cb06f67029e350 app/viewModels/fileC.js 1'></a> &nbsp;<b><a href='https://github.com/Jumio/con-rest/pull/73#discussion_r29734837'>File: app/viewModels/fileC.js:L1-57</a></b>
- <a href='https://github.com/EnoF'></a> typo, I'll amend the commit
- <a href='https://github.com/dominikkukacka'></a> already did that :)
- [ ] <a href='#crh-comment-Pull 268468bd3a04209ac6e2dc2af2cb06f67029e350 callback.js 20'></a> &nbsp;<b><a href='https://github.com/Jumio/con-rest/pull/73#discussion_r29734983'>File: callback.js:L1-37</a></b>
- <a href='https://github.com/EnoF'></a> I'll clean this up in an additional commit
- [ ] <a href='#crh-comment-Pull 268468bd3a04209ac6e2dc2af2cb06f67029e350 callback.js 1'></a> &nbsp;<b><a href='https://github.com/Jumio/con-rest/pull/73#discussion_r29735080'>File: callback.js:L1-37</a></b>
- <a href='https://github.com/EnoF'></a> What is the use of an additional server?
- <a href='https://github.com/dominikkukacka'></a> that's just for testing the execution of requests against an endpoint
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/Jumio/con-rest/pull/73#issuecomment-87972414'>General Comment</a></b>
- <a href='https://github.com/EnoF'></a> code review :+1:

<a href='https://www.codereviewhub.com/Jumio/con-rest/pull/73?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/Jumio/con-rest/pull/73?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/Jumio/con-rest/pull/73?approve=0'></a>&nbsp;<a href='https://github.com/Jumio/con-rest/pull/73'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by EnoF** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/Jumio/con-rest/pull/73'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by EnoF** [comment]
code review :+1:

**Comment by TheBay0r** [comment]
:shipit:

File: app/viewModels/fileC.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+(function fielCScope(angular) {
```
Comment (EnoF) [inline app/viewModels/fileC.js]: typo, I'll amend the commit

File: callback.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  'strict': true,
+  'inflate': true,
+  'limit': '5mb',
+  'type': 'json'
+}));
+
+
+['get', 'post', 'put', 'options'].forEach(function(method){
+  app[method](/.*/g, function(req, res) {
+    console.log('url:', req.url);
```
Comment (EnoF) [inline callback.js]: I'll clean this up in an additional commit

File: callback.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+var express = require('express');
```
Comment (EnoF) [inline callback.js]: What is the use of an additional server?

Comment (dominikkukacka) [inline callback.js]: that's just for testing the execution of requests against an endpoint

File: app/viewModels/fileC.js (hunk 1 above)
Comment (dominikkukacka) [inline app/viewModels/fileC.js]: already did that :)

**Comment by EnoF** [comment]
:shipit:

**Pull request merged**